### DISCUSSION TITLE
Upload Limit Attachment’s

### DISCUSSION THREAD

**Discussion by Fabian1410** [Budibase/budibase#7389] [Help]
Is there a reason why the file upload limit is hardcoded to 20mb? Why isn't it possible to configure it? https://github.com/Budibase/budibase/blob/612ef21e75d7265f2df20666a602f0f494d1754b/packages/bbui/src/Form/Core/Dropzone.svelte#L18

**Comment by R2bEEaton** [comment]
I think this is probably for continuity between the Cloud version and the self-hosted. If this is self-hosted, I totally think this should be modifiable up to wherever the internal limitations are.

```
export let fileSizeLimit = {MaxUploadFileSize}
```

For the time being, you could fork the repository and make a change as you see fit if you plan to or are self-hosting, but an official option should totally be made down the line. It could be in the `docker-compose.yaml` for example. Looks like there's a similar issue open that you can follow. #5587

**Reply by Fabian1410** [reply]
yes, or in admin settings in the app …
